manager/integration/tests/common.py

```diff
@@ -6270,3 +6274,58 @@ def wait_delete_dm_device(api, name):
             break
         time.sleep(RETRY_INTERVAL)
     assert not found
+
+
+def set_tags_for_node_and_its_disks(client, node, tags): # NOQA
```

Comment:
This is not writing by me
It is used in test_ha long time ago

My test cases and `test_ha` test cases don't have such needs to set flexible tags.

But I agree with you
We can have another util function for this flexibility.
I can add another one, but lets keep the original one, because I don't want to change the `test_ha` testing code in this PR.